File: javascript/py/99 Bottles of Beer.py
bottles = 99
while bottles >= 1:
    print(f"{bottles} bottles of beer on the wall, 99 bottles of beer.")
    bot = bottles - 1
    print(
        f"Take one down and pass it around, {bottles} bottles of beer on the wall.\n")
    if bottles == 0:
        print(f"No more bottles of beer on the wall, no more bottles of beer.")
        print(f"Go to the store and buy some more, 99 bottles of beer on the wall.\n")

File: javascript/99bottles of beer.js
function beer() {
    for (var i = 99; i >= 2;) {
        console.log(i + " bottles of beer on the wall, " + i + " bottles of beer.");
        i--
        console.log("Take one down and pass it around, " + i + " bottles of beer on the wall.")
        if (i === 1) {
            console.log(i + " bottle of beer on the wall, " + i + " bottle of beer.")
            console.log("Take one down and pass it around, no more bottles of beer on the wall.")
        }
    }
}
beer();